### PR TITLE
Fixes bug 1143724 - Used Super Search to power the Reports tab of rep…

### PR DESCRIPTION
**Pull request by AdrianGaudebert**
…ort/list.

@peterbe r?

That's another one of those commits that once were in master but got removed. It's the same content but since I had to do intense conflict resolution, I guess it would be good to give a quick look.

### CODE REVIEW

**Comment by peterbe** [comment]
A "quick look" he says to a 300 line diff :)

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report_list_reports.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ Panels.register('reports', function() {
 
 var Columns = (function() {
     var COLUMNS = null;
-    var KEY = 'report-list-report-columns';
+    var KEY = 'report-list-reports-columns';
```
Comment (peterbe) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report_list_reports.js]: I remember :)

File: webapp-django/crashstats/crashstats/views.py [hunk 2]
```diff
@@ -1382,36 +1445,40 @@ def report_list(request, partial=None, default_context=None):
...
                 break
 
+        api_columns = [x['key'] for x in context['columns']]
+        # The correlations tab requires some keys to always be there. Since
+        # they are low cost, we add them here, to keep the advantages of the
+        # shared cache between the two tabs.
+        api_columns += [
+            'platform', 'version', 'release_channel', 'install_time', 'date'
+        ]
+
```
Comment (peterbe) [inline webapp-django/crashstats/crashstats/views.py]: rm

**Comment by peterbe** [comment]
Well, it works :)

What I don't like is that it says "Url" instead of "URL". And "Uuid", "Cpu arch", "Json dump" and "Productid" to mention a few. Also, why does it show "Productid" twice? Is that because `"productid".title() == `"ProductID".title()`?

File: webapp-django/crashstats/crashstats/views.py (hunk 2 above)
Comment (AdrianGaudebert) [inline webapp-django/crashstats/crashstats/views.py]: rm? Why should I remove this?

**Comment by AdrianGaudebert** [comment]
Yeah the fact that it appears twice is due to the fact that we expose the field from the processed crash and from the raw crash. I suppose we could change the Super Search Fields list to not exposed the one in the processed crash?

File: webapp-django/crashstats/crashstats/views.py (hunk 2 above)
Comment (peterbe) [inline webapp-django/crashstats/crashstats/views.py]: oops. my bad. I didn't see that it was actually used.

**Comment by peterbe** [comment]
>  I suppose we could change the Super Search Fields list to not exposed the one in the processed crash? 

+1

**Comment by AdrianGaudebert** [comment]
I will come back to this later!

**Pull request closed without merging**